packages/module/patternfly-docs/content/extensions/component-groups/about-component-groups.md

```diff
@@ -4,14 +4,27 @@ id: About component groups
 sortValue: 1
 sourceLink: https://github.com/patternfly/react-component-groups/blob/main/packages/module/patternfly-docs/content/extensions/component-groups/about-component-groups.md
 --- 
+import { SectionGallery } from '@patternfly/documentation-framework/components/sectionGallery/sectionGallery';
+import galleryData from './gallery-data.json';
 
 Component groups lives in its own package [`@patternfly/react-component-groups`](https://www.npmjs.com/package/@patternfly/react-component-groups) 
 
 # Component groups
 
-The component groups extension contains a range of React components that are more complex than basic PatternFly components. These component groups combine and adjust multiple base components to provide opinionated solutions for recurring use cases across products using PatternFly. 
+The component groups extension contains a range of React components that are more complex than basic PatternFly components. These component groups combine and adjust multiple base components to provide opinionated solutions for recurring use cases across products using PatternFly.
 
-The creation and maintenance of this extension is a collaborative effort between the PatternFly and Red Hat Hybrid Cloud Console teams. All component groups receive accessibility and design reviews, and we are working to improve internationalization, add unit test coverage, and ensure that CSS overrides all correctly reference the PatternFly CSS API.
+The creation and maintenance of this extension is a collaborative effort between the PatternFly and Red Hat Hybrid Cloud Console teams. All component groups receive accessibility and design reviews, and we are working to improve internationalization, add test coverage, and ensure that CSS overrides all correctly reference the PatternFly CSS API.
+
+Below, you can see a gallery of categories representing the components:
+
+<SectionGallery
+  section="Component groups"
+  galleryItemsData={galleryData}
+  placeholderText="Search component groups by name"
```

Comment:
```suggestion
  placeholderText="Search component groups by name"
```
maybe we can just remove the search bar for now, since there's not a lot to filter through? but if that's a lot of work, it's doing no harm by being there!

Reply:
@edonehoo the gallery component does not allow it yet. We may want to talk to @nicolethoen about the possibility of adding that configuration.